Makefile

```diff
@@ -14,6 +19,13 @@ lint: get_deps
 test: get_deps
 	yarn test
 
+test_integration:
```

Comment:
This looks dependent on `setup_gitmodules` /cc @erezmus  @norisalsaadie

Reply:
setup_gitmodules is usually done manually or as part of setup (in reviewer repo).